### ISSUE TITLE
Languages & Linters could be merged

### ISSUE THREAD

**Issue by viatsko** [viatsko/awesome-vscode#12]
Some languages are already providing linters, probably we could merge them - I'm also thinking about table format in that section

**Comment by SamVerschueren** [comment]
Care to provide an example of how you would show it in a table?

**Comment by viatsko** [comment]
I'm currently thinking on
Syntaxes, Linters, Snippets, Other
columns and let the same packages be on the same level in the first two columns (as we might end up of duplicating a lot of packages in Syntax & Linters category as a lot of them are actually doing two things at once)

**Comment by viatsko** [comment]
Agree, I've started expanding the list into sections since many languages got "plugins ecosystem" around them during these years. Will close once I'm done with this.

**Comment by mr-ubik** [comment]
Also grouping all snippets package for a language under one section would be better ie:

---

## Python
- Python Ext
### Linters
### Snippets
#### Flask
#### Django

---

Or something like

## Python
- Language-specific packages
### Flask
- Any Flask related package
### Django
- Any[ Django related package

**Comment by duttaditya18** [comment]
What about Linters that lint more than one language? How would we deal with that?
